scripts/gen_vimdoc.py

```diff
@@ -83,8 +89,8 @@
 
 
 # DEBUG = ('DEBUG' in os.environ)
-INCLUDE_C_DECL = ('INCLUDE_C_DECL' in os.environ)
-INCLUDE_DEPRECATED = ('INCLUDE_DEPRECATED' in os.environ)
+INCLUDE_C_DECL = bool(int(os.environ.get('INCLUDE_C_DECL', '0') or False))
+INCLUDE_DEPRECATED = bool(int(os.environ.get('INCLUDE_DEPRECATED', '0') or False))
```

Comment:
Is the `or False` needed here since you have the default value?

```suggestion
INCLUDE_C_DECL = bool(int(os.environ.get('INCLUDE_C_DECL', '0')))
INCLUDE_DEPRECATED = bool(int(os.environ.get('INCLUDE_DEPRECATED', '0')))
```

Reply:
Could this also be written as `os.environ.get('INCLUDE_C_DECL') != '0'`? Imo that is clearer than wrapping in `bool(int())`.

Reply:
Fixed, the one you suggested is much better. thanks!